Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest/Profiles/MRTK-Quest_CameraProfile.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: cd071e4825d7028469978568a7f2dc33
```

Comment:
Are these the same GUIDs from the original repo? We probably want to regenerate them, so there aren't GUID clashes for anybody who already has the original extension installed when they update MRTK (or installs later).

Reply:
How do you regenerate guids?